keras/applications/convnext.py

```diff
@@ -218,7 +218,8 @@ def __init__(self, init_values, projection_dim, **kwargs):
 
     def build(self, input_shape):
         self.gamma = tf.Variable(
```

Comment:
Can you replace this with self.add_weight(), which the more standard way to create weights for layer? (it has more logics for handling other aspects.)

Reply:
Sure. Made a new commit with the suggested change.